.gitignore

```
@@ -49,18 +49,12 @@ client/windows-universal-js/ZUMOAPPNAME.sln
 client/windows-universal-js/ZUMOAPPNAME/ZUMOAPPNAME.Shared/js/default.js
 client/windows-universal-js/ZUMOAPPNAME/ZUMOAPPNAME.Windows/packages.config
 client/windows-universal-js/ZUMOAPPNAME/ZUMOAPPNAME.WindowsPhone/packages.config
-client/xamarin.android/ZUMOAPPNAME.sln
 client/xamarin.android/ZUMOAPPNAME/packages.config
-client/xamarin.android/ZUMOAPPNAME/ZUMOAPPNAME.csproj
-client/xamarin.iOS/ZUMOAPPNAME.sln
 client/xamarin.iOS/ZUMOAPPNAME/packages.config
-client/xamarin.iOS/ZUMOAPPNAME/ZUMOAPPNAME.csproj
 backend/dotnet/Quickstart/ZUMOAPPNAMEService/packages.config
 backend/dotnet/Quickstart/ZUMOAPPNAMEService/ZUMOAPPNAMEService.csproj
 /backend/dotnet/Quickstart/ZUMOAPPNAMEService/Web.config
-client/xamarin.forms/*.sln
 client/xamarin.forms/**/*.config
-client/xamarin.forms/**/*.csproj
 client/xamarin.forms/**/project.json
 client/xamarin.forms/Droid/Resources/Resource.designer.cs
-/client/android/ZUMOAPPNAME/app/build.gradle
+/client/android/ZUMOAPPNAME/app/build.gradle
```

It would be nice if we can make paths consistent (with or without `/`) in this file.
```suggestion
client/android/ZUMOAPPNAME/app/build.gradle
```